powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java

```diff
@@ -43,6 +43,8 @@
 import java.util.*;
 import java.util.concurrent.TimeUnit;
 
+import static com.wultra.security.powerauth.client.model.enumeration.UserActionResult.APPROVAL_FAILED;
```

Comment:
I prefer to isolate test constants from production constants. When production code changes, this test does not fail.